app/scripts/directives/metas/metas-actividades.js

```diff
@@ -300,16 +300,18 @@ angular
               })
               itemactividad["registro_plan_adquisiciones-actividad"].forEach(function (item){
                 var actcont = 0;
-                var movi = $scope.movimiento.find(function(mov){
+                var movi = $scope.movimiento.filter(function(mov){
                   return mov.datos.ActividadId === item.actividad.Id;
                 });
-                if(movi != undefined){
+                if(movi.length != 0 ){
```

Comment:
Es posible que esto llegue a disparar una exepción si movi.length llega a estar indefinido. Quizás baste con:
```suggestion
                if(movi && movi.length){
```